python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -2058,6 +2058,65 @@ def testBackground( self ) :
 		r.option( "ai:background", None )
 		self.assertEqual( arnold.AiNodeGetPtr( options, "background" ), None )
 
+	def testVDBs( self ) :
+
+		import IECoreVDB
+
+		tmpFile = self.temporaryDirectory() + "/test.ass"
+
+		r = GafferScene.Private.IECoreScenePreview.Renderer.create(
+			"Arnold",
+			GafferScene.Private.IECoreScenePreview.Renderer.RenderType.SceneDescription,
+			tmpFile
+		)
+
+		attributes = IECore.CompoundObject( {
+			"ai:volume:velocity_scale" : IECore.FloatData( 10 ),
+			"ai:volume:velocity_fps" : IECore.FloatData( 25 ),
+			"ai:volume:velocity_outlier_threshold" : IECore.FloatData( 0.5 ) } )
+
+		# Camera needs to be added first as it's being used for translating the VDB.
+		# We are doing the same when translating actual Gaffer scenes here:
+		# https://github.com/GafferHQ/gaffer/blob/master/src/GafferScene/Render.cpp#L254
+		r.camera(
+			"testCamera",
```

Comment:
Although this is adding a camera, it's not actually declaring it as the active render camera using the ["camera" option](https://github.com/GafferHQ/gaffer/blob/master/include/GafferScene/Private/IECoreScenePreview/Renderer.h#L105). See [`testRenderRegion()`](https://github.com/GafferHQ/gaffer/blob/master/python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py#L105) for an example of how to do that.

If you take a look at the ass file this test generates, you'll see that it contains an additional default camera which has been generated automatically due to the lack of the "camera" option. And crucially, it's this default camera which is the one that appears in the arnold options. And what's more, it has `shutter_start == shutter_end == 0`!